references/detection/train_tpu.py

```diff
@@ -133,6 +133,7 @@ def do_prediction(image_path, use_cpu=False):
     model.eval()
 
     image = Image.open(image_path)
+    image = torchvision.transforms.functional.resize(image, (800, 600))
```

Comment:
FYI, once we get to training we'll need to do this exact thing in the input pipeline (but I believe it should already be there).

Reply:
Sure thing, this is really for prototyping, long term I think this should go somewhere like [`GeneralizedRCNNTransform.resize()`](https://github.com/jysohn23/vision/blob/b45cdbf/torchvision/models/detection/transform.py#L55) where they already do some resizing based on min/max size thresholds.